core/src/main/java/hudson/util/XStream2.java

```diff
@@ -207,7 +207,7 @@ protected Converter createDefaultConverter() {
      * Specifies that a given field of a given class should not be treated with laxity by {@link RobustCollectionConverter}.
      * @param clazz a class which we expect to hold a non-{@code transient} field
      * @param field a field name in that class
-     * @since TODO
+     * @since 2.85 this method can be used from outside core, before then it was restricted since initially added in 1.551 / 1.532.2
```

Comment:
Grammatical train wreck, suggestions welcome.

Reply:
I was able to parse it